Branch-SDK/src/io/branch/referral/Defines.java

```diff
@@ -65,7 +64,23 @@
         IsFirstSession("+is_first_session"),
         AndroidDeepLinkPath("$android_deeplink_path"),
         DeepLinkPath("$deeplink_path"),
-        AndroidAppLinkURL("android_app_link_url");
+
+        AndroidAppLinkURL("android_app_link_url"),
+
+        ContentScreen("content_screen"),
+        ContentPath("content_path"),
```

Comment:
What is the suggested usage of `content_screen` and `content_path`, and if they're special params, why isn't there a `$` in front of them?